drivers/sensor/bosch/bmp388/bmp388.c

```diff
@@ -577,26 +578,27 @@ static int bmp388_init(const struct device *dev)
 #define BMP388_INT_CFG(inst)
 #endif
 
-#define BMP388_INST(inst)						   \
-	static struct bmp388_data bmp388_data_##inst = {		   \
-		.odr = DT_INST_ENUM_IDX(inst, odr),			   \
-		.osr_pressure = DT_INST_ENUM_IDX(inst, osr_press),	   \
-		.osr_temp = DT_INST_ENUM_IDX(inst, osr_temp),		   \
-	};								   \
-	static const struct bmp388_config bmp388_config_##inst = {	   \
-		BMP388_BUS_CFG(inst)					   \
-		BMP388_INT_CFG(inst)					   \
-		.iir_filter = DT_INST_ENUM_IDX(inst, iir_filter),	   \
-	};								   \
-	PM_DEVICE_DT_INST_DEFINE(inst, bmp388_pm_action);		   \
-	SENSOR_DEVICE_DT_INST_DEFINE(					   \
-		inst,							   \
-		bmp388_init,						   \
-		PM_DEVICE_DT_INST_GET(inst),				   \
-		&bmp388_data_##inst,					   \
-		&bmp388_config_##inst,					   \
-		POST_KERNEL,						   \
-		CONFIG_SENSOR_INIT_PRIORITY,				   \
-		&bmp388_api);
-
-DT_INST_FOREACH_STATUS_OKAY(BMP388_INST)
+#define BMP388_INST(inst, chipid)\
```

Comment:
Again renaming would make it morr obvious what you are doing here

Reply:
BMP3XX_INST() now.